src/services/analytics.ts

```diff
@@ -192,15 +191,22 @@ type SignInFinish = {
 // Settigns
 type SettingsChangeTheme = {
   eventName: Name.SETTINGS_CHANGE_THEME
-  mode: 'dark' | 'light' | 'auto'
+  mode: 'dark' | 'light' | 'matrix' | 'auto'
 }
-type SettignsStartTwitterOauth = {
+type SettingsStartTwitterOauth = {
   eventName: Name.SETTINGS_START_TWITTER_OAUTH
 }
 type SettingsCompleteTwitterOauth = {
   eventName: Name.SETTINGS_COMPLETE_TWITTER_OAUTH
   is_verified: boolean
 }
+type SettingsStartInstagramOauth = {
+  eventName: Name.SETTINGS_START_INSTAGRAM_OAUTH
```

Comment:
wanna capture handle here too? would be nice to see who's doing it and maybe not getting to the complete event